frontend/src/views/activity/Activity.vue

```diff
@@ -74,7 +64,45 @@ Displays a single activity
         </v-btn>
 
         <pdf-download-button-nuxt :config="printConfig()" class="ml-3" />
-        <pdf-download-button-react :config="printConfig()" />
+        <pdf-download-button-react :config="printConfig()" class="ml-3" />
+
+        <!-- hamburger menu -->
+        <v-menu offset-y>
+          <template #activator="{ on, attrs }">
+            <v-btn icon v-bind="attrs" v-on="on">
+              <v-icon>mdi-dots-vertical</v-icon>
+            </v-btn>
+          </template>
+
+          <v-list>
+            <!-- layout/content switch (switch to layout mode) -->
+            <v-list-item v-if="!layoutMode" :disabled="!isContributor" @click="layoutMode = true">
```

Comment:
minor:
I would keep the button here.
For me it was confusing that the button back is not anymore in the hamburger menu

Reply:
Both ok for me. Let's quickly vote in the next meeting

Reply:
## Core Meeting Decision
Hide menu in layoutmode and show a button to exit layout mode instead.